data/raw/keybindings.json

```diff
@@ -1138,6 +1138,19 @@
     },
     {
         "type" : "keybinding",
+        "id": "QUIT",
+        "category": "NEW_CHAR_DESCRIPTION",
```

Comment:
It is intended. The world creation window works the same way. It has a separate entry for the tab where you input the name.